amarna/amarna.py

```diff
@@ -153,7 +153,10 @@ def analyze_directory(rootdir: str, rule_names: List[str]) -> List[Any]:
                 all_results += amarna.run_local_rules(fname, parsed_cairo_file)
                 amarna.run_gatherer_rules(fname, parsed_cairo_file)
 
-    all_results += amarna.run_post_process_rules()
+    try:
+        all_results += amarna.run_post_process_rules()
+    except Exception:
```

Comment:
This would hide any other potential bug that we have and only print that it couldn't find any other cairo files. I would add a boolean after we ran the gatherer rules and check here if that variable is true.

Reply:
Sure, using boolean instead of catch-all exception handling [2989f79](https://github.com/crytic/amarna/pull/39/commits/2989f7979e162313627a5a54eeed94a363709fc3)